pdns/rec_channel_rec.cc

```diff
@@ -1195,6 +1259,36 @@ static StatsMap toProxyMappingStatsMap(const string& name)
   return entries;
 }
 
+static StatsMap toRemoteLoggerStatsMap(const string& name)
+{
+  const auto pbasename = getPrometheusName(name);
+  StatsMap entries;
+
+  auto stats1 = broadcastAccFunction<RemoteLoggerStats_t>(pleaseGetRemoteLoggerStats);
+  auto stats2 = broadcastAccFunction<RemoteLoggerStats_t>(pleaseGetOutgoingRemoteLoggerStats);
+  auto stats3 = broadcastAccFunction<RemoteLoggerStats_t>(pleaseGetFramestreamLoggerStats);
+  uint64_t count = 0;
+  for (const auto& [stats, type] : {make_pair(stats1, "protobuf"), make_pair(stats2, "outgoingProtobuf"), make_pair(stats3, "dnstapFrameStream")}) {
+    for (const auto& [key, entry] : stats) {
+      auto keyname = pbasename + "{address=\"" + key + "\",type=\"" + type + "\",count=\"";
```

Comment:
clang-tidy says `+` is inefficient and might incur unnecessary allocations. If it's important here, use `+=` or `append()` instead.